src/cryptoadvance/specter/device_manager.py

```diff
@@ -96,3 +100,15 @@ def remove_device(self, device, wallet_manager=None):
         if isinstance(device, BitcoinCore):
             device.delete(wallet_manager)
         self.update()
+
+    @property
+    def devices_backup_file(self):
+        memory_file = BytesIO()
+        with zipfile.ZipFile(memory_file, 'w') as zf:
+            for device in self.devices.values():
+                data = zipfile.ZipInfo('{}.json'.format(device.name))
+                data.date_time = time.localtime(time.time())[:6]
+                data.compress_type = zipfile.ZIP_DEFLATED
+                zf.writestr('{}.json'.format(device.name), device.json)
+        memory_file.seek(0)
+        return memory_file
```

Comment:
I am not sure it should be a property, better use a function.
When I am using property somewhere I assume that it's fast, so I can access it many times without carying much about caching.
But reading all files and zipping them may not be fast.